python-sdk/rebuff/rebuff_pythonsdk.py

```diff
@@ -0,0 +1,228 @@
+import secrets
+from typing import Any, Optional, Tuple, Union
+from detect_pi_heuristics import detect_prompt_injection_using_heuristic_on_input
+from detect_pi_vectorbase import init_pinecone, detect_pi_using_vector_database
+from detect_pi_openai import render_prompt_for_pi_detection, call_openai_to_detect_pi
+from pydantic import BaseModel
+import langchain
+
+
+class Rebuff_Detection_Response(BaseModel):
+    heuristic_score: float
+    openai_score: float
+    vector_score: float
+    run_heuristic_check: bool
+    run_vector_check: bool
+    run_language_model_check: bool
+    max_heuristic_score: float
+    max_model_score: float
+    max_vector_score: float
+    injection_detected: bool
+
+
+class RebuffPython:
```

Comment:
I believe you'll need to include this class and `RebuffDetectionResponse` in `python-sdk/rebuff/__init__.py` in order for those classes to be imported from outside this module.